lib/Github/Api/Organization/Members.php

```diff
@@ -46,6 +46,15 @@ public function conceal($organization, $username)
         return $this->delete('orgs/'.rawurlencode($organization).'/public_members/'.rawurlencode($username));
     }
 
+    /*
+     * Add user to organization
+     */
+    
```

Comment:
no extra newline after the phpdoc, and no extra space character on the end of the function definition